01/app.js

```diff
@@ -1 +1,9 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+const element = document.getElementsByClassName('comments__item' && 'comments__item--newest');
+
+
+if(element) {
```

Comment:
Ponieważ wyszukujemy wiele elementów to nawet pusta tablica jest "prawdą".
Należałoby sprawdzić ilość tych elementów.